example/rhizome/dataset/0ea1fc7a-dd97-4adc-a10e-169c6597bcde.ttl

```diff
@@ -0,0 +1,43 @@
+@prefix area: <https://ontology.okp4.space/thesaurus/area/> .
+@prefix core: <https://ontology.okp4.space/core/> .
+@prefix dataset: <https://ontology.okp4.space/dataverse/dataset/> .
+@prefix dataspace: <https://ontology.okp4.space/dataverse/dataspace/> .
+@prefix dstmeta: <https://ontology.okp4.space/metadata/dataset/> .
+@prefix license: <https://ontology.okp4.space/thesaurus/license/> .
+@prefix mediatype: <https://ontology.okp4.space/thesaurus/media-type/> .
+@prefix meta: <https://ontology.okp4.space/metadata/> .
+@prefix service: <https://ontology.okp4.space/dataverse/service/> .
+@prefix topic: <https://ontology.okp4.space/thesaurus/topic/> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+meta:8de7ca33-378e-445c-b087-f5ef334d2cca a dstmeta:GeneralMetadata ;
+    core:describes dataset:0ea1fc7a-dd97-4adc-a10e-169c6597bcde ;
+    core:hasCreator "Anses"^^xsd:string ;
+    core:hasDescription "Avec 3 185 aliments référencés, cette nouvelle version de la table Ciqual intègre le détail de tous les sucres individuels contenus dans les aliments. Elle propose également des données actualisées sur les principaux fruits et légumes, y compris ceux cultivés en Outre-mer. Pour être en adéquation avec les nouvelles pratiques alimentaires des français, la table inclut désormais une cinquantaine de nouveaux aliments adaptés aux régimes végétariens."@fr ;
```

Comment:
Please, add translation for `@en` and `@fr` as we did with cec4e7dae8361caec72e89cdcebaf86d5f3b2c47 for the other dataset instances.